Documentation/network/servicemesh/mutual-authentication/mutual-authentication-example.rst

```diff
@@ -0,0 +1,235 @@
+    Serial number     : 110848236251310359782141595494072495768
+
+    SPIFFE ID         : spiffe://spiffe.cilium/spire/agent/k8s_psat/default/d4a8a6da-d808-4993-b67a-bed250bbc53e
+    Attestation type  : k8s_psat
+    Expiration time   : 2023-07-04 18:39:55 +0000 UTC
+    Serial number     : 7806033782886940845084156064765627978
+
+Notice that the SPIRE Server uses Kubernetes Projected Service Account Tokens (PSATs) to verify the identity of a SPIRE Agent running on a Kubernetes Cluster. 
+Projected Service Account Tokens provide additional security guarantees over traditional Kubernetes Service Account Tokens and when supported by a Kubernetes cluster, PSAT is the recommended attestation strategy.
+
+Verify SPIFFE identity
+======================
+
+Now that we know the SPIRE service is healthy, let's verify that the Cilium and SPIRE integration has been successful:
+
+- The Cilium agent and operator should have a registered delegate identity with the SPIRE Server.
+- The Cilium operator should have registered identities with the SPIRE server on behalf of the workloads (Kubernetes Pods).
+
+Verify that the Cilium agent and operator have identities on the SPIRE server:
+
+.. code-block:: shell-session
+
+    $ kubectl exec -n cilium-spire spire-server-0 -c spire-server -- /opt/spire/bin/spire-server entry show -parentID spiffe://spiffe.cilium/ns/cilium-spire/sa/spire-agent
+    Found 2 entries
+    Entry ID         : b6424c87-4323-4d64-98dd-cd5b51a1fcbb
+    SPIFFE ID        : spiffe://spiffe.cilium/cilium-agent
+    Parent ID        : spiffe://spiffe.cilium/ns/cilium-spire/sa/spire-agent
+    Revision         : 0
+    X509-SVID TTL    : default
+    JWT-SVID TTL     : default
+    Selector         : k8s:ns:kube-system
+    Selector         : k8s:sa:cilium
+
+    Entry ID         : 8aa91d65-16c4-48a0-bc1f-c9bf26e6a25f
+    SPIFFE ID        : spiffe://spiffe.cilium/cilium-operator
+    Parent ID        : spiffe://spiffe.cilium/ns/cilium-spire/sa/spire-agent
+    Revision         : 0
+    X509-SVID TTL    : default
+    JWT-SVID TTL     : default
+    Selector         : k8s:ns:kube-system
+    Selector         : k8s:sa:cilium-operator
+
+
+Verify that the workloads have identities registered with the SPIRE server.
+
+First, let's get the Cilium Identity of the *echo* Pod first as the SPIFFE ID - that uniquely identifies a workload - is partially based on the Cilium identity.
+
+.. code-block:: shell-session
+
+    $ IDENTITY_ID=$(kubectl get cep -l app=echo -o=jsonpath='{.items[0].status.identity.id}')
+    $ echo $IDENTITY_ID
+    17947
+
+The SPIFFE ID for this workload is based on the ``spiffe://spiffe.cilium/identity/$IDENTITY_ID`` format.
+
+Verify that this Pod has a registered identity on the SPIRE server:
+
+.. code-block:: shell-session
+
+    # kubectl exec -n cilium-spire spire-server-0 -c spire-server -- /opt/spire/bin/spire-server entry show -spiffeID spiffe://spiffe.cilium/identity/$IDENTITY_ID
```

Comment:
Some of your commands use `$` for a prompt (good), this one uses `#` which indicates it should be run as root (but is it the case?), some below don't have a prompt at all. I know we're not consistent with prompts in the docs in general, but it would be nice to be consistent at least in a new page we add. Could you please harmonise them (`#` for command that should be run as root, `$` everywhere else)?

Reply:
Don't think root is needed anywhere - corrected to `$` everywhere.